models/equiformer_v3/test_equiformer_v3_discovery.py

```
@@ -0,0 +1,286 @@
+
+                self.relax_results[material_id] = {
+                    "structure": structure,
+                    "energy": energy,
+                }
+            except Exception:
+                print(f"Failed to relax {material_id}")
+                output_err_filename = (
+                    f"error_index@{self.job_index}_total@{self.num_jobs}.txt"
+                )
+                with open((self.output_path / output_err_filename), "a") as txt_file:
+                    txt_file.write(f"{mat_id}\n")
```

Ruff (0.15.9)

[error] Undefined name `mat_id`

(F821)